packages/react-components/react-components/src/index.ts

```diff
@@ -1885,3 +1885,88 @@ export type {
   PresenceMotionFn,
   PresenceComponentProps,
 } from '@fluentui/react-motion';
+
+export {
+  CarouselButton,
+  carouselButtonClassNames,
+  renderCarouselButton_unstable,
+  useCarouselButtonStyles_unstable,
+  useCarouselButton_unstable,
+  CarouselNav,
+  carouselNavClassNames,
+  renderCarouselNav_unstable,
+  useCarouselNavStyles_unstable,
+  useCarouselNav_unstable,
+  CarouselNavButton,
+  carouselNavButtonClassNames,
+  renderCarouselNavButton_unstable,
+  useCarouselNavButtonStyles_unstable,
+  useCarouselNavButton_unstable,
+  Carousel,
+  carouselClassNames,
+  renderCarousel_unstable,
+  useCarouselStyles_unstable,
+  useCarousel_unstable,
+  CarouselCard,
+  carouselCardClassNames,
+  renderCarouselCard_unstable,
+  useCarouselCardStyles_unstable,
+  useCarouselCard_unstable,
+  CarouselAutoplayButton,
+  carouselAutoplayButtonClassNames,
+  renderCarouselAutoplayButton_unstable,
+  useCarouselAutoplayButtonStyles_unstable,
+  useCarouselAutoplayButton_unstable,
+  CarouselNavImageButton,
+  carouselNavImageButtonClassNames,
+  renderCarouselNavImageButton_unstable,
+  useCarouselNavImageButtonStyles_unstable,
+  useCarouselNavImageButton_unstable,
+  CarouselSlider,
+  carouselSliderClassNames,
+  renderCarouselSlider_unstable,
+  useCarouselSliderStyles_unstable,
+  useCarouselSlider_unstable,
+  type CarouselSliderProps,
+  type CarouselSliderSlots,
+  type CarouselSliderState,
```

Comment:
this was added by our generator or by hand ? if by generator can you please fill a bug ? otherwise please move it to `export type` bucket

Reply:
Hi @Hotell - I've reproduced this on main and seems it happens consistently, have filed a bug here: https://github.com/microsoft/fluentui/issues/32968

I'll move these manually into the correct type export